crates/hdk/src/prelude.rs

```diff
@@ -71,6 +76,7 @@ pub use holochain_zome_types::post_commit::PostCommitCallbackResult;
 pub use holochain_zome_types::query::ActivityRequest;
 pub use holochain_zome_types::query::AgentActivity;
 pub use holochain_zome_types::query::ChainQueryFilter as QueryFilter;
+pub use holochain_zome_types::query::ChainQueryFilter;
```

Comment:
I don't think we should have both of these in the prelude?

Reply:
@freesig this was entirely accidental but i'm personally ok with having both the literal import and the alias :thinking: